src/app/donation-start/donation-start-form/donation-start-form-new.component.html

```diff
@@ -476,45 +477,67 @@
             {{ stripeError }}
           </p>
 
+          <p></p>
+
+          <table class="summary" *ngIf="donation">
+            <tr class="donationAmount">
+              <td>Your donation to {{ campaign.charity.name }}</td>
+              <td>{{ donationAmount | exactCurrency:donation.currencyCode }}</td>
+            </tr>
+            <tr *ngIf="donation.matchedAmount > 0">
+              <td>Matched Amount</td>
+              <td>{{ expectedMatchAmount() | exactCurrency:donation.currencyCode }}</td>
+            </tr>
+            <tr *ngIf="giftAidAmount() > 0">
+              <td>Gift Aid Amount</td>
+              <td>{{ giftAidAmount | exactCurrency:donation.currencyCode }}</td>
+            </tr>
+            <tr *ngIf="feeCoverAmount() > 0 || (creditPenceToUse === 0 && !campaign.feePercentage)">
+              <td>Platform fee</td>
```

Comment:
Noticing that the screenshot shoews platform fee £0 - I don't think we want it to show platform fee with any current campaigns, not sure why it's showing up. This logic I think is copied from what we had already, so not sure exacltly what's going on - or why the second part of the condition exists.

Reply:
Yeah, this definitely looks wrong. It looks like a previous containing element probably had the full condition clause in its `ngIf` and it got moved to the wrong place, as it looks like a combination of what should be
`[the case where fee shows] || [the case where tip shows]`

Reply:
Right. Fixed in 4fc0a17c .